plugins/kotlin/j2k/shared/tests/testData/newJ2k/function/extendsBaseWhichExtendsObject.kt

```diff
@@ -42,7 +42,7 @@ internal open class Base : Cloneable {
     }
 
     @Throws(Throwable::class)
-    protected open fun finalize() {
+    open fun finalize() {
```

Comment:
I think 'protected' should be preserved for 'finalize' that directly overrides built-in 'finalize'. I will fix this in `JavaStandardMethodsConversion` before merging this PR.